idaes/core/solvers/tests/test_petsc.py

```diff
@@ -26,6 +26,15 @@
 from idaes.core.solvers.features import dae
 
 def rp_example():
+    """This example is done multiple ways to test a few common errors where the
+    integrator and fully time-discretized problem differ and alternative working
+    formulations.
+
+    The PETSc utilities raise an exception when a differential variable is fixed.
+    While this is okay for the fully time-discretized problem, for the integrator
+    it implies the derivative is 0.  While this problem can be solved with the
```

Comment:
I suggest changing to: "While this is okay for the fully time-discretized problem, the integrator will not correctly link a fixed differential variable (at non-initial time points) with a time derivative."

Reply:
The issue is that the derivative should not necessarily be zero. As long as this is corrected, I'm fine.

Reply:
Sounds good.  Updated wording.